### PR TITLE
Remove the "Beta" badge, if the parent or technology is "Beta"

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 93785405

## Summary

Removes the Beta badge from children of Beta items in the navigator. If a whole technology is marked as Beta, none of the navigator items show as Beta.

## Dependencies

NA

## Testing

Use an archive with Beta items or Beta technology

Steps:
1. Assert that a Beta technology does not mark any of it's children as Beta
2. Assert that a Beta parent, does not mark it's children as Beta.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Comment by dobromir-hristov** [comment]
@swift-ci test

File: src/components/Navigator.vue [hunk 1]
```diff
@@ -189,6 +189,10 @@ export default {
           // push child to parent
           parent.childUIDs.push(node.uid);
         }
+        if (node.beta && ((parent && parent.beta) || this.technology.beta)) {
```
Comment (mportiz08) [inline src/components/Navigator.vue:192]: Can you add a quick comment explaining the reasoning for doing this?

**Review by mportiz08 — APPROVED** [review]

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Comment by dobromir-hristov** [comment]
@swift-ci test

**Pull request merged**